src/pages/ImportPage.tsx

```diff
@@ -4,23 +4,20 @@ import NamespacedPage from '../components/NamespacedPage/NamespacedPage';
 import PageLayout from '../components/PageLayout/PageLayout';
 import { useQuickstartCloseOnUnmount } from '../hooks/useQuickstartCloseOnUnmount';
 import { getQueryArgument } from '../shared/utils';
-import { useWorkspaceInfo } from '../utils/workspace-context-utils';
+import { useApplicationBreadcrumbs } from '../utils/breadcrumb-utils';
 
 const ImportPage: React.FunctionComponent = () => {
   useQuickstartCloseOnUnmount();
 
-  const { workspace } = useWorkspaceInfo();
   const applicationName = getQueryArgument('application');
+  const applicationBreadcrumbs = useApplicationBreadcrumbs();
```

Comment:
I know we were not already showing the current application in the breadcrumb, but shouldn't we?

Reply:
removed application context for import page route based on Christian's comment https://github.com/openshift/hac-dev/pull/406#discussion_r1103021441.  If we name the application as `import` then it will conflict with the details page routes.

Reply:
I think you can still add application name to the breadcrumb without the route param. You can use the query param for application name in breadcrumbs. That way user can see the application in which he is going to add the component.
<img width="460" alt="Screenshot 2023-02-16 at 3 52 27 PM" src="https://user-images.githubusercontent.com/6041994/219338877-ea187688-c8b8-4c17-8883-947ee9dd324b.png">

Reply:
Added the application breadcrumb in component editing page.